Assets/HoloToolkit/Input/Scripts/KeywordManager.cs

```diff
@@ -84,26 +102,30 @@ void OnDestroy()
             }
         }
 
+        /// <summary>
+        /// Listens for key presses and calls the corrisponding keyword event
+        /// </summary>
         private void ProcessKeyBindings()
         {
-            foreach (var kvp in KeywordsAndResponses)
+            for (int i = 0; i < KeywordsAndResponses.Length; i++)
```

Comment:
Even encouraged to use for instead of foreach in [Unity Performance Considerations](https://developer.microsoft.com/en-us/windows/holographic/performance_recommendations_for_unity)